cmd/nerdctl/container_run_network_linux_test.go

```diff
@@ -49,12 +51,24 @@ func TestRunInternetConnectivity(t *testing.T) {
 		{
 			args: []string{"--net", customNet},
 		},
+		{
+			args:               []string{"--net", netutil.NetworkID(customNet)},
+			dockerIncompatible: true,
+		},
+		{
+			args:               []string{"--net", netutil.NetworkID(customNet)[:12]},
+			dockerIncompatible: true,
+		},
 		{
 			args: []string{"--net", "host"},
 		},
 	}
 	for _, tc := range testCases {
 		tc := tc // IMPORTANT
+		if tc.dockerIncompatible {
+			// docker compute id algorithm is different from nerdctl
+			testutil.DockerIncompatible(t)
```

Comment:
This should be compatible if you inspect the id via `docker network inspect`

Reply:
fixed